packages/components/src/molecules/RadioGroup/RadioOption.tsx

```diff
@@ -23,7 +23,7 @@ export interface RadioOptionProps
 
 const RadioOption = forwardRef<HTMLInputElement, RadioOptionProps>(
   function RadioOption(
-    { label, value, children, testId = 'store-radio-option', ...otherProps },
+    { label, value, children, testId = 'fs-radio-option', ...otherProps },
```

Comment:
@eduardoformiga do you think we can rename it to be more specific to this context?

something like `fs-radio-group-option`

And I think we can also rename the data-attr of this component.
We are currently using `data-fs-radio-option`.

If you do so, remember to add this data-attr, to the `Customization` section. 😉

Reply:
Nice! done [here](https://github.com/vtex/faststore/pull/1604/commits/497c062a099441df64a5efc8cf88a2afb659299f).